src/ugeneui/html/version_news_en.html

```diff
@@ -21,10 +21,12 @@
 <h3>Changes in UGENE v. 45</h3>
 
 <ul style="margin-top: 10px; margin-bottom: 10px;">
-
-    <li style="margin-bottom: 10px;"> This is a development version of UGENE.</li>
-    <li style="margin-bottom: 10px;"> The list of changes will be added to the release version.</li>
-
+    <li style="margin-bottom: 10px;">New visualization mode in MSA Editor: Wrap Mode.</li>
+    <li style="margin-bottom: 10px;">
+        Phylogenetic tree visualization improvements: hiding nodes, copying of tree image to clipboard, branch curvature.
+    </li>
+    <li style="margin-bottom: 10px;">Primer 3 tool is upgraded to the latest version.</li>
```

Comment:
Usually tools are _updated_, not upgraded, but maybe I'm too picky

Reply:
Fixed